package.json

```diff
@@ -24,5 +24,8 @@
   "bugs": {
     "url": "https://github.com/wasm-tool/rust-loader/issues"
   },
-  "homepage": "https://github.com/wasm-tool/rust-loader#readme"
+  "homepage": "https://github.com/wasm-tool/rust-loader#readme",
+  "devDependencies": {
+    "console-emoji": "0.0.2"
```

Comment:
I have to say that this dependency scares me a bit (https://www.npmjs.com/package/console-emoji).

Reply:
Don't worry. I'll remove it.

Reply:
done!